typings/restify-validator/restify-validator.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for adding validator to restify
```

Comment:
The main thing is that I want it to be a separate file (so that we can grab updates to `express-validator.d.ts` without thinking too hard), but if what you're suggesting is to put it in the same directory, I'm ok with that. Suggestions for names? `express-to-restify-validator`? :)